application/src/data/brand/tokens.ts

```diff
@@ -0,0 +1,68 @@
+/**
+ * Brand tokens and design system values
+ *
+ * This file contains the core brand identity tokens including colors, typography,
+ * spacing, and other design system values. Colors are fetched from
+ * @hoverkraft-tech/branding repository at build time when available.
+ */
+
+export interface ColorToken {
+  name: string;
+  hex: string;
+  rgb: string;
+  usage: string;
+}
+
+export interface TypographyToken {
+  family: string;
+  weights: number[];
+  usage: string;
+  fallback?: string;
+}
+
+export interface LogoAsset {
+  name: string;
+  path: string;
+  formats: string[];
+  usage: string;
+  minimumSize?: string;
+  clearSpace?: string;
+}
```

Comment:
move these types into a dedicated file so they can be imported here and by `generated-*.ts` files too